ovh/resource_cloud_project_kube_test.go

```diff
@@ -974,7 +969,7 @@ func TestAccCloudProjectKubeVRack(t *testing.T) {
 	}
 	configData2 := configData{
 		ServiceName:                    serviceName,
-		VrackID:                        vrackID,
+		VlanID:                         vlanID,
 		Name:                           name,
 		Region:                         region,
 		DefaultVrackGateway:            "10.4.0.1",
```

Comment:
```suggestion
		DefaultVrackGateway:            "",
```
Let's the DHCP server gives the GatewayIP.
(ATM, the KubeVRack with a gateway is not working, the provided DefaultVrackGateway is not matching the deployed one)